w2w/w2w.py

```diff
@@ -508,24 +500,87 @@ def wrf_remove_urban(
     dst_data.to_netcdf(info.dst_nu_file)
 
 
-# Make WRF grid info available for Resampler (tmp file)
-def create_wrf_gridinfo(info: Info) -> None:
+# Get WRF grid info for Resampler
+# Inspired by: https://fabienmaussion.info/2018/01/06/wrf-projection/
+def _get_wrf_grid_info(info: Info) -> Dict[str, Dict[str, Any]]:
```

Comment:
This should be:
```suggestion
def _get_wrf_grid_info(info: Info) -> Dict[str, Any]:
```
That's how the dict looks like:
```python
{
    'crs': '+proj=eqc +lat_ts=0 +lat_0=0 +lon_0=5.84999990463257 +x_0=0 +y_0=0 '
        '+R=6370000 +units=m +no_defs',
    'transform': Affine(1111.7747802734375, 0.0, -838835.115342933,
       0.0, 1111.7747802734375, 4577176.609447704)
}
```